migration/sql-files/062-categories.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	name text NOT NULL
+);
+
+CREATE INDEX categories_id_index ON categories (id);
+CREATE UNIQUE INDEX categories_name_idx ON categories (name) WHERE deleted_at IS NULL;
+
+CREATE TABLE workitemtype_categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	category_id uuid NOT NULL,
+	workitemtype_id uuid NOT NULL
```

Comment:
I will rename this field for consistency to `work_item_type_id` as we did for `link_type_id` and others